k8s/daemonset_envfrom.yaml

```diff
@@ -11,7 +11,7 @@ spec:
       serviceAccountName: scalyr-service-account
       containers:
       - name: scalyr-agent
-        image: scalyr/scalyr-k8s-agent-profiling:2.0.47.profiling.8
+        image: scalyr/scalyr-k8s-agent-profiling:2.0.48.profiling.9
```

Comment:
Note: Image is not pushed yet so for my own testing I edited it.

Reply:
As long we keep increasing the version number every build (even if it was just for testing) it's fine.  Don't be tempted just to overwrite it :D